packages/xo-server/config.toml

```diff
@@ -42,6 +42,12 @@ perMessageDeflate = { threshold = 524288 } # 512kiB
 defaultTokenValidity = '30 days'
 maxTokenValidity = '0.5 year'
 
+# When a user logs in using an authentication provider, but another user with the
+# same username is found in XO:
+# - if true: the 2 users will be merged
+# - if false: the user will not be able to log in.
+mergeProvidersUsers = true
```

Comment:
AFAIU, this behavior is especially useful for the transition period, should we had a comment to remove this in a year?

Reply:
It will also be useful after that, to transition from "users being created manually" to "users being created by the plugin". The plugin will need to know if a native XO user that has the same username as a LDAP user can be considered as the same user.